.eslintrc.json

```diff
@@ -42,6 +42,10 @@
         "allowSingleLine": true
       }
     ],
+    "linebreak-style":[
```

Comment:
I also added this and it's really a life-saver when it comes to time-saving haha.. :)